assets/src/blocks/CarouselHeader/CarouselHeaderBlock.js

```diff
@@ -48,6 +48,6 @@ export const registerCarouselHeaderBlock = () =>
     edit: CarouselHeaderEditor,
     save: frontendRendered(BLOCK_NAME),
     deprecated: [
-      { ...attributes, ...carouselHeaderV1 }
```

Comment:
Removed `attributes` from here since they are already included in `carouselHeaderV1`